src/pages/MemoriesPage.js

```diff
@@ -6,6 +6,8 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 import DeleteIcon from '../assets/images/icons/delete.png'
 import cam from '../assets/images/icons/cam.png'
+import search from '../assets/images/icons/search.png'
+import { useState } from 'react'
 
```

Comment:
Maybe you could organize your imports with cmd+shift+p :)

Reply:
I am going to organize it when I clean up the code XD